_observability-plugins/data-prepper/get-started.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Get Started
+parent: Data Prepper
+nav_order: 1
+---
+
+# Get started with Data Prepper
+
+Data Prepper is an independent component, not an OpenSearch plugin, that converts data for use with OpenSearch. It's not bundled with the all-in-one OpenSearch installation packages.
```

Comment:
This section gives the impression that Data Prepper only works with OpenSearch, consider rephrasing.

Reply:
If we had separate documentation and website I would agree with your suggestion. However I am inclined to leave the existing language as is. Looking at the OpenSearch documentation website as a whole I think this language make the most sense given our reader is viewing Data Prepper from an OpenSearch perspective.